src/main.js

```diff
@@ -0,0 +1,105 @@
+const clear = document.querySelector('.clear');
+const dateElement = document.getElementById('date');
+const list = document.getElementById('list');
+const input = document.getElementById('input');
+
+const CHECK = 'fa-check-circle';
+const UNCHECK = 'fa-circle-thin';
+const LINE_THROUGH = 'lineThrough';
+
+let LIST; let
```

Comment:
- [ ] OPTIONAL 
It's a best/ standard practice to reserve full capitalization of variable names only for true constants. Kindly use a lowercase for your `let` variable